src/components/sbb-datepicker-previous-day/sbb-datepicker-previous-day.tsx

```diff
@@ -161,6 +177,25 @@ export class SbbDatepickerPreviousDay implements ComponentInterface, ButtonPrope
     return this._dateAdapter.today();
   }
 
+  private async _setAriaLabel(): Promise<void> {
```

Comment:
not sure if we should externalize this function to avoid code duplication between next and previous day buttons?